### PR TITLE
Sb 24273 : LMS functional test env variables

### PR DESCRIPTION
**Pull request by neha0305verma**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-24273" title="SB-24273" target="_blank">StorySB-24273</a>  Platform-Courses: Enable functional test run
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: kubernetes/ansible/roles/lms-ft/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+- name: copy the template file
+  template:
+    src: "{{item}}.j2"
+    dest: "/tmp/lms-ft.env.j2"
+    mode: 0755
+  with_items:
+    - lms-ft.env
+  when: enable_lms-ft == "yes"
```
Comment (SMYALTAMASH) [inline kubernetes/ansible/roles/lms-ft/tasks/main.yml:9]: we don't need this condition.

File: kubernetes/ansible/roles/lms-ft/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+- name: inject the lms-ft into apimanager pod
+  shell: kubectl patch deployments.apps -n "{{namespace}}" apimanager --patch="$(cd /tmp | source lms-ft.env | mvn -X clean verify)"
```
Comment (SMYALTAMASH) [inline kubernetes/ansible/roles/lms-ft/tasks/main.yml:12]: please replace this line with below
please check the command to execute

File: kubernetes/ansible/roles/lms-ft/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  when: enable_lms-ft == "yes"
```
Comment (SMYALTAMASH) [inline kubernetes/ansible/roles/lms-ft/tasks/main.yml:13]: we don't need this condition

File: kubernetes/ansible/roles/lms-ft/templates/lms-ft.env.j2 [hunk 4]
```diff
@@ -0,0 +1,56 @@
+sunbird_username=ntptest104
```
Comment (SMYALTAMASH) [inline kubernetes/ansible/roles/lms-ft/templates/lms-ft.env.j2:1]: can you remove these dummy values and variablize them?

Comment (neha0305verma) [reply kubernetes/ansible/roles/lms-ft/templates/lms-ft.env.j2:1]: where I can set those variables??

**Comment by rjshrjndrn** [comment]
@neha0305verma what's this PR for? I've checked the mentioned Jira issue. It has no comments. Can you please update that?

**Comment by neha0305verma** [comment]
> @neha0305verma what's this PR for? I've checked the mentioned Jira issue. It has no comments. Can you please update that?

Updated the ticket.

**Pull request closed without merging**